### ISSUE TITLE
Not supported on Android 6.0 API - bug

### ISSUE THREAD

**Issue by szczepcio95** [naoufal/react-native-touch-id#85]
While catching error on TouchId.isSupported I'm getting info that device doesn't support fingerprint authorization even though Android is supporting such feature from api level 23(6.0)

**Comment by zibs** [comment]
I think I've run this on 6.0 before but I can't quite remember if that's totally true. 

What version of the lib are you using and have you linked everything correctly? 

In any case, this PR #70 fixes it for 5 & < so you could use this PR for now until it is merged in.

**Comment by szczepcio95** [comment]
We've added 

```
if (!fingerprintManager.isHardwareDetected()) {
          return false; 
      }
```

in FingerprintAuthModule.java and problem is solved

**Comment by zibs** [comment]
Just merged #91 which adds this.

Thanks for reporting. will be out in next release.

**Comment by zibs** [comment]
released in 4.0.1
